dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -228,6 +254,24 @@ export const YoutubeBlockComponent = ({
 					isMainMedia={isMainMedia}
 				/>
 			)}
+
+			{showOverlayCaption && (
```

Comment:
What if `hideCaption` is true in this component? Are we still going to show the caption?